### DISCUSSION TITLE
Hiding certain messages

### DISCUSSION THREAD

**Discussion by AKGoggles** [ORelio/Minecraft-Console-Client#2448] [Q&A]
Is there a way to hide specific messages without putting "ShowSystemMessages = false"?

My chat is being spammed with one message that I can only turn off by doing that. There are other messages also being shut off by hiding system message that I still want to be able to see, so being able to just hide the one message would be perfect.

**Comment by ReinforceZwei** [comment]
Use chat filter option.
